src/libopensc/card-openpgp.c

```diff
@@ -3466,7 +3473,7 @@ sc_get_openpgp_driver(void)
 	pgp_ops.list_files	= pgp_list_files;
 	pgp_ops.get_challenge	= pgp_get_challenge;
 	pgp_ops.read_binary	= pgp_read_binary;
-	pgp_ops.write_binary	= pgp_write_binary;
+	pgp_ops.write_binary	= NULL;
```

Comment:
clang-format suggested the following change:


```
-	pgp_ops.write_binary	= NULL;
+	pgp_ops.write_binary = NULL;
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-openpgp.c -->